src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8881,6 +8881,7 @@ namespace UnitarySystems {
             }
 
             if (MoistureLoad < LatOutputOn && this->m_DehumidControlType_Num == DehumCtrlType::CoolReheat) {
+                HXUnitOn = true; // HX is needed to meet moisture load
```

Comment:
And for CoolReheat the HX needs to be turned on when sensible only load control can't meet the latent load.